joomla/joomla-cms
# [4.3] Guided Tours replace deprecated access of input property

### Summary of Changes

Apply changes from #39029 for guided tours

### Testing Instructions

- save a tour
- open tour edit form
- open step edit form
- check module (mod_guidedtours)

### Actual result BEFORE applying this Pull Request

Use of deprecated access to the application's input property

### Expected result AFTER applying this Pull Request

All works like before, but without using deprecated access to the application's input property

### Link to documentations
Please select:
